tools/benchmark_visualization.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+from pathlib import Path
+import os
+
+"""
+This script can be used to visualize benchmark results. The script assumes multiple benchmark 
+results in csv files under the build directory. These results can be generated with the 
+Graaf_perf executable:
+
+build/perf/Graaf_perf --benchmark_out=benchmark_results.csv --benchmark_out_format=csv
+"""
+
```

Comment:
It's an idiomatic [best practice](https://stackoverflow.com/a/419185) to wrap your executed code into `if __name__ == '__main__' ` to avoid unintended side effects. In you're case, it won't make a difference, but it's a good practice nevertheless